src/odemis/driver/xt_client.py

```diff
@@ -417,7 +423,7 @@ def pump(self):
             self.server.pump()
 
     def get_vacuum_state(self):
-        """Returns: (string) the vacuum state of the microscope chamber to see if it is pumped or vented."""
+        """Returns: ("vacuum" or "vented") the vacuum state of the microscope chamber to see if it is pumped or vented."""
```

Comment:
Well, it can also return "prevac", "pumping", "venting", and "vacuum_error". Please mention them too.